packages/db-states-file/lib/states/statesInMemFileDB.js

```diff
@@ -339,6 +347,16 @@ class StatesInMemoryFileDB extends InMemoryFileDB {
         }
         this.dataset[id] = data;
 
+        // If data === undefined, the state was just created and not filled with value
+        if (data !== undefined) {
+            setImmediate(() => {
+                const event = { val: null, binary: true, size: data.byteLength, ack: true };
```

Comment:
We need to Adjust Typings in any case for this!!

Reply:
Ich habe nicht mal eine Idee, wo das sein konnte

Reply:
In extra package, I tagged the issue, we will handle that later. @AlCalzone normally takes care of these :-) Because we need to adjust when we release not too much before